libs/primitives/src/lib.rs

```diff
@@ -49,6 +49,14 @@ pub mod types {
 	// Ensure that origin is either Root or fallback to use EnsureOrigin `O`
 	pub type EnsureRootOr<O> = EitherOfDiverse<EnsureRoot<AccountId>, O>;
 
+	/// The technical fellowship collective which can whitelist proposal for the
+	/// WhitelistedCaller track
+	pub type TechnicalCollective = pallet_collective::Instance2;
+
+	/// The technical membership which handles membership of the
+	/// TechnicalCollective. It is not linked to the WhitelistedCaller track.
+	pub type TechnicalMembership = pallet_membership::Instance1;
```

Comment:
Could they go in `runtime/common/src/lib.rs` under `instances`?

Reply:
I am not sure whether this makes sense. I added these two instances here because the existing `CouncilCollective` is also defined in `primitives::types`. We cannot move it out because the `AllOfCouncil`, `HalfOfCouncil` and similar types of `primitives::types` are dependent on `CouncilCollective` and `primitives` should not have `common` as dependency.

I can imagine that we might eventually define similar types (`AllOfTechnical...` etc.) for the `TechnicalCollective` which should probably stay in `primitives::types`?

Reply:
I would move the current `AllOfCouncil` and `CouncilCollective` to `runtime-common`. For me, primitives should not know about pallets or how runtimes are configured. WDYT?

Reply:
That's what I did in [f530ee559c6a0575826ec67780cbb426d7e54f16](https://github.com/centrifuge/centrifuge-chain/pull/1828/commits/f530ee559c6a0575826ec67780cbb426d7e54f16#diff-8930d436577b42479d8367c28afd439e7bb65a9f14ed350201685ba0602701b4). I apologize for the difficulty to follow this properly because of my continuous force rebases. I just wanted to have the Centrifuge OpenGov commit as clean as possible so I can cherry-pick it in a month or so and have as few conflicts as possible.

Reply:
Ah cool! I didn't see it. No worries! Thanks